src/vs/workbench/contrib/experiments/common/experimentService.ts

```diff
@@ -276,9 +326,9 @@ export class ExperimentService extends Disposable implements IExperimentService
 			});
 			return Promise.all(promises).then(() => {
 				type ExperimentsClassification = {
-					experiments: { classification: 'SystemMetaData', purpose: 'FeatureInsight' };
+					experiments: { classification: 'SystemMetaData', purpose: 'FeatureInsight'; };
 				};
-				this.telemetryService.publicLog2<{ experiments: IExperiment[] }, ExperimentsClassification>('experiments', { experiments: this._experiments });
+				this.telemetryService.publicLog2<{ experiments: IExperiment[]; }, ExperimentsClassification>('experiments', { experiments: this._experiments });
```

Comment:
Are these types supposed to have semicolons?

Reply:
I think this was done by the TS language features formatting